src/theme/NavbarItem/NavbarNavLink.tsx

```diff
@@ -6,6 +6,13 @@ import { isRegexpStringMatch } from '@docusaurus/theme-common';
 import IconExternalLink from '@theme/Icon/ExternalLink';
 import type { Props } from '@theme/NavbarItem/NavbarNavLink';
 
+const GITHUB_STARS_SESSION_STORAGE_NAME = 'openfga_github_stars';
+
+interface githubStarsSessionStorage {
```

Comment:
Can you capitalize the interface name? e.g.

```suggestion
interface GithubStarsSessionStorage {
```

so it is not mistaken for a variable